themes/custom/campaign_base/scripts/custom/campaign_base.js

```diff
@@ -69,7 +69,7 @@
         var $thisImage = $container.find('img');
 
         var imgSrc = $thisImage.attr('data-src');
-        var imgSrcSet = $thisImage.attr('srcset');
+        var imgSrcSet = $thisImage.attr('src');
```

Comment:
Does this still work okay for browsers that **do** use srcset?

Reply:
We need `data-src` to deal with images loaded via blazy and `src` to other images. `srcset` is not necessary because it is in all `<source>` of pictures and all `<picture>` have a `<img src>`.

Reply:
Coolcool, was too tired to think properly on Friday, right you are